src/FredData.jl

```diff
@@ -25,7 +25,7 @@ const FIRST_REALTIME     = Date(1776,07,04)
 const LAST_REALTIME      = Date(9999,12,31)
 const EARLY_VINTAGE_DATE = "1991-01-01"
 const FRED_DATE_FORMAT   = DateFormat("yyyy-mm-dd HH:MM:SSzz")
-const OUTPUT_TZ          = TimeZone("UTC")
+const OUTPUT_TZ          = UTC
```

Comment:
Our usage of TimeZones has changed. Previously we were "switching time zones" to UTC. Could convert to *any* time zone. Now we are "converting date/time without time zone information" (re TimeZones.jl docs). Could only to convert to either UTC or machine's local time. Can you see how this is slightly different?

Anyways, can you please rename the constant to `OUTPUT_TZ_TYPE` to reflect the change in API usage?